### PR TITLE
Add `MiniscopeBehaviorInterface`

### PR DESCRIPTION
**Pull request by weiglszonja**
Adds `MiniscopeBehaviorInterface` for Miniscope behavioral data.

- Uses [ndx-miniscope](https://github.com/catalystneuro/ndx-miniscope) to add a `Miniscope` device with the behavioral camera metadata, and an `ImageSeries` in external mode that is linked to the device. 
- More info about the behavioral data stream is at https://github.com/catalystneuro/ndx-miniscope

- [x] Remove ndx-miniscope pin after https://github.com/catalystneuro/ndx-miniscope/pull/11 is merged

### CODE REVIEW

File: src/neuroconv/datainterfaces/behavior/miniscope/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ndx-miniscope @ git+https://github.com/catalystneuro/ndx-miniscope.git@7de541ebceca7e73571ad7aecafb3ec6b8a19c16#egg=ndx-miniscope
```
Comment (CodyCBakerPhD) [inline src/neuroconv/datainterfaces/behavior/miniscope/requirements.txt:1]:
```suggestion
ndx-miniscope>=0.5.1
```

**Comment by CodyCBakerPhD** [comment]
@weiglszonja Ready for final review?

**Comment by weiglszonja** [comment]
> @weiglszonja Ready for final review?

Yes, thank you!

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
@weiglszonja LGTM! Excited to see the final `MiniscopeConverter` after this

**Pull request merged**